src/main/kotlin/no/nav/familie/ef/sak/task/BehandlingsstatistikkTask.kt

```diff
@@ -0,0 +1,151 @@
+import no.nav.familie.kontrakter.ef.felles.StønadType
+import no.nav.familie.kontrakter.ef.iverksett.BehandlingsstatistikkDto
+import no.nav.familie.kontrakter.ef.iverksett.Hendelse
+import no.nav.familie.kontrakter.felles.oppgave.Oppgave
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import org.springframework.stereotype.Service
+import java.time.ZoneId
+import java.util.Properties
+import java.util.UUID
+
+@Service
+@TaskStepBeskrivelse(
+        taskStepType = BehandlingsstatistikkTask.TYPE,
+        beskrivelse = "Sender behandlingsstatistikk til iverksett"
+)
+
+class BehandlingsstatistikkTask(private val iverksettClient: IverksettClient,
+                                private val behandlingService: BehandlingService,
+                                private val fagsakService: FagsakService,
+                                private val søknadService: SøknadService,
+                                private val vedtakService: VedtakService,
+                                private val oppgaveService: OppgaveService,
+                                private val personService: PersonService
+) : AsyncTaskStep {
+    private val zoneIdOslo = ZoneId.of("Europe/Oslo")
+
+    override fun doTask(task: Task) {
+        val (behandlingId, hendelse, hendelseTidspunkt, gjeldendeSaksbehandler, oppgaveId) = objectMapper.readValue<BehandlingsstatistikkTaskPayload>(
+                task.payload
+        )
+
+        val personIdent = behandlingService.hentAktivIdent(behandlingId)
+        val behandling = behandlingService.hentBehandling(behandlingId)
+        val fagsak = fagsakService.hentFagsak(behandling.fagsakId)
+
+        val sisteOppgaveForBehandling = finnSisteOppgaveForBehandlingen(behandlingId, oppgaveId)
+        val resultatBegrunnelse = finnResultatBegrunnelse(hendelse, behandlingId)
+        val søker = personService.hentSøker(personIdent);
+        val søknadstidspunkt = finnSøknadstidspunkt(fagsak, behandlingId)
+
+        val behandlingsstatistikkDto = BehandlingsstatistikkDto(
+                behandlingId = behandlingId,
+                personIdent = personIdent,
+                gjeldendeSaksbehandlerId = gjeldendeSaksbehandler,
+                eksternFagsakId = fagsak.eksternId.id.toString(),
+                hendelseTidspunkt = hendelseTidspunkt.atZone(zoneIdOslo),
+                søknadstidspunkt = søknadstidspunkt.atZone(zoneIdOslo),
+                hendelse = hendelse,
+                behandlingResultat = behandling.resultat.name,
+                resultatBegrunnelse = resultatBegrunnelse,
+                opprettetEnhet = sisteOppgaveForBehandling.opprettetAvEnhetsnr ?: "9999",
+                ansvarligEnhet = sisteOppgaveForBehandling.tildeltEnhetsnr ?: "9999",
+                strengtFortroligAdresse = søker.adressebeskyttelse.gjeldende()?.gradering == AdressebeskyttelseGradering.STRENGT_FORTROLIG,
```

Comment:
Burde denne sjekke for `STRENGT_FORTROLIG_UTLAND` og?

Reply:
Jepp